Source/WebCore/rendering/RenderLayer.h

```diff
@@ -147,6 +147,7 @@ struct ScrollRectToVisibleOptions {
     const ScrollAlignment& alignY { ScrollAlignment::alignCenterIfNeeded };
     ShouldAllowCrossOriginScrolling shouldAllowCrossOriginScrolling { ShouldAllowCrossOriginScrolling::No };
     ScrollBehavior behavior { ScrollBehavior::Auto };
+    OnlyAllowForwardScrolling onlyAllowForwardScrolling { OnlyAllowForwardScrolling::No };
```

Comment:
We're going to add a  bit here for autoscrolling soon, so might as we use an OptionSet<> here, and `ShouldAllowCrossOriginScrolling` can be in it.

Reply:
I will do this in a follow up patch so that I can get this escape fix in.